src/PublicSuffixListManager.php

```diff
@@ -35,6 +35,15 @@ class PublicSuffixListManager
     const PRIVATE_PSL_PHP_FILE = 'private-public-suffix-list.php';
 
     /**
+     * @var PublicSuffixList Public Suffix List
```

Comment:
Comment doesn't match variable.

Reply:
changed ... but after the merge I think the naming may still change. I used a temporary wording.